### ISSUE TITLE
Cannot activate camera when ringing is in progress

### ISSUE THREAD

**Issue by szotsaki** [signalapp/Signal-Android#10776]
- [X] I have searched open and closed issues for duplicates
- [X] I am submitting a bug report for existing functionality that does not work as intended
- [X] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [X] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
It's not possible to activate the camera when an audio call is started and ringing is in progress.

### Steps to reproduce

1. Start an audio call to another signal member
2. While it's ringing try to turn on camera

**Actual result:** Black background appears on the calling screen.
**Expected result:** Camera picture appears on the calling screen.

### Device info

**Device:** Sony XPERIA X
**Android version:** 8.0.0
**Signal version:** 5.1.9

### Link to debug log
https://debuglogs.org/16bae44ae308751788fb1ca4af32906916a59331b100da937a7e8c370ea63e0f

**Comment by szotsaki** [comment]
Yes, still.

Freshened debug logs: https://debuglogs.org/38151895ae1ce519c730472343425b152cac6a4ae939c8dc1ed4dcf7137e9ceb

**Comment by szotsaki** [comment]
Still an issue.

https://debuglogs.org/android/5.34.5/b421a343064afa77f89c399f083c0275ccc0a2c0ddcef1fd8f698e1fe40d0ed8

**Comment by cody-signal** [comment]
Yup, you right. Do you do this often? I fully acknowledge it's a bug and should behave as you describe, just curious if it hinders you a lot over just starting a video call out of the gate.

**Comment by szotsaki** [comment]
No, it doesn't hinder me at all. I think I discovered it quite a while ago with some other call-related issues but since then I think I never "encountered" this bug (except for now when the bot pinged me).

**Comment by UnreachableCode** [comment]
Hi @szotsaki and @cody-signal is this still an issue? Not able to reproduce on 7.43.1 on a Pixel 6, Android 14.
